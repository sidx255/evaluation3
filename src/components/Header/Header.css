
.header {
    display: flex;
    font-size: 2.0em;
    text-align: left;
    align-items: center;
    height: 2em;
    color: rgb(255, 255, 255);
    background-color: #000000;
}

.header-text {
    display: flex;
    padding-left: 1.5em;
    font-size: 2.0em;
    font-family: 'Big Shoulders Inline Display', cursive;
}