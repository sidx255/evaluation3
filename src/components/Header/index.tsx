import React from 'react';

import './Header.css';

const Header = () => {
  return (
    <div className='header' data-testid='header'>
      <p className='header-text'>EVENTIFY</p>
    </div>
  );
};

export default Header;