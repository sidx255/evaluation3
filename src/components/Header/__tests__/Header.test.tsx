import React from 'react';
import { render } from '@testing-library/react';
import Header from '../index';

test('renders Header', () => {
  render(<Header />);
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});