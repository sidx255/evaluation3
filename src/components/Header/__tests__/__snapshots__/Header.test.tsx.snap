// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header 1`] = `
<DocumentFragment>
  <div
    class="header"
    data-testid="header"
  >
    <p
      class="header-text"
    >
      EVENTIFY
    </p>
  </div>
</DocumentFragment>
`;
